core/src/main/kotlin/behavior/channel/threads/ThreadParentChannelBehavior.kt

```diff
@@ -33,7 +34,7 @@ interface ThreadParentChannelBehavior : TopGuildMessageChannelBehavior {
      * [terminal operators](https://kotlinlang.org/docs/reference/coroutines/flow.html#terminal-flow-operators) instead.
 
      */
-    val activeThreads: Flow<ThreadChannel> get() = supplier.getActiveThreads(id)
+    val activeThreads: Flow<ThreadChannel> get() = guild.activeThreads.filter { it.parentId == id }
```

Comment:
if you're delegating calls to other entities, you should make sure they're using the same supply strategy